build-tools/automation/yaml-templates/run-msbuild-tests.yaml

```diff
@@ -3,7 +3,7 @@ parameters:
   jobName:                # Name of the job
   jobDisplayName:         # Display name of the job
   agentCount:             # Number of build agents to run in parallel
-  testFilter:             # Filter used to select tests (NUnit test selection language, not dotnet test filter language)
+  testFilter: ' '         # Filter used to select tests (NUnit test selection language, not dotnet test filter language)
```

Comment:
What was the reason to make this default to a space character?

Reply:
We still pass the parameter name to the tool when the string is empty, which was failing:

> Required argument missing for option: '--test-filter'.

Using some whitespace felt like the simplest fix.  We could alternatively update the parameter to require the `--test-filter` parameter name, or [duplicate this block](https://github.com/xamarin/xamarin-android/blob/main/build-tools/automation/yaml-templates/run-sliced-nunit-tests.yaml#L9-L17) and not pass the filter param to one of the invocations if either of those seem better?

Reply:
Maybe the space is simpler, and put a note in the comment?

Reply:
I ended up updating this to be more explicit to avoid any potential confusion about the param, and in case the slicer tool changes.